src/index.html

```diff
@@ -7,6 +7,145 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <!-- <div class="calendar">
+      <div class="calendar__container">
+        <div class="calendar__row1">
+          <span  class="calendar__row1span1 cell">
+
+          </span>
+        </div>
+        <div class="calendar__row2">
+          <span  class="calendar__row2span1 cell">
+
+          </span>
+          <span  class="calendar__row2span2 cell">
+
+          </span>
+          <span  class="calendar__row2span3 cell">
+
+          </span>
+          <span  class="calendar__row2span4 cell">
+
+          </span>
+          <span  class="calendar__row2span5 cell">
+
+          </span>
+          <span  class="calendar__row2span6 cell">
+
+          </span>
+          <span  class="calendar__row2span7 cell">
+
+          </span>
+        </div>
+        <div class="calendar__row3">
+          <span  class="calendar__row3span1 cell">
+
+          </span>
+          <span  class="calendar__row3span2 cell">
+
+          </span>
+          <span  class="calendar__row3span3 cell">
+
+          </span>
+          <span  class="calendar__row3span4 cell">
+
+          </span>
+          <span  class="calendar__row3span5 cell">
+
+          </span>
+          <span  class="calendar__row3span6 cell">
+
+          </span>
+          <span  class="calendar__row3span7 cell">
+
+          </span>
+        </div>
+        <div class="calendar__row4">
+          <span  class="calendar__row4span1 cell">
+
+          </span>
+          <span  class="calendar__row4span2 cell">
+
+          </span>
+          <span  class="calendar__row4span3 cell">
+
+          </span>
+          <span  class="calendar__row4span4 cell">
+
+          </span>
+          <span  class="calendar__row4span5 cell">
+
+          </span>
+          <span  class="calendar__row4span6 cell">
+
+          </span>
+          <span  class="calendar__row4span7 cell">
+
+          </span>
+        </div>
+        <div class="calendar__row5">
+          <span  class="calendar__row5span1 cell">
+
+          </span>
+          <span  class="calendar__row5span2 cell">
+
+          </span>
+          <span  class="calendar__row5span3 cell">
+
+          </span>
+          <span  class="calendar__row5span4 cell">
+
+          </span>
+          <span  class="calendar__row5span5 cell">
+
+          </span>
+          <span  class="calendar__row5span6 cell">
+
+          </span>
+          <span  class="calendar__row5span7 cell">
+
+          </span>
+        </div>
+      </div>
+    </div> -->
+
+    <!-- calendar__start-day--sun calendar__month-lenght--31 -->
+
+    <div class="calendar">
+      <div class="calendar__container">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day-mon calendar--month-length-30 ">
```
Do you really need a container? Also, I don't see the modifiers. How can I modify the calender?
![image](https://user-images.githubusercontent.com/96409563/210315401-bb483015-f3a2-4ba5-ba8b-e124e6a2e197.png)